site/cypress/e2e/Cypress-UI/pronouns_change.spec.js

```diff
@@ -27,20 +27,21 @@ describe('Tests cases abut changing user pronouns', () => {
         cy.login('student');
 
         //change back to old pronouns
-        cy.get('#pronouns_val').click();
+        cy.get('#pronouns_val').click({force: true});
 
         //set to default
-        cy.get('button[aria-label="Clear pronoun input"]').click(); //clear input using trash can
+        cy.get('button[aria-label="Clear pronoun input"]').click({force: true}); //clear input using trash can
 
         //set to false
-        cy.get('#pronouns-forum-display').uncheck();
+        cy.get('#pronouns-forum-display').uncheck({force: true});
+        cy.get('#edit-pronouns-submit').click({force: true});
 
-        cy.get('#edit-pronouns-submit').click();
+        //Wait for HTML content to update
+        cy.wait(10000);
```

Comment:
Is there a reason for such a long delay? If you are waiting for the page to update, would cy.waitPageChange() be a better option (defined in cypress/support/commands.js) If not, could you try and find something on the page that should be different and assert the presence/absence of the item with a longer timeout instead of a long wait?